CHANGES/603.misc

```diff
@@ -1 +1 @@
-Update the development environment to Python 3.8 and base image to ubi8:8.2
\ No newline at end of file
+Update the development environment to Python 3.8, update base image to ubi8:8.2 and update requirements
```

Comment:
Since you are editing this changelog snippet, may want to change to say ubi8:8.4 if that separate PR gets approved/merged. And may be worth removing the "development" piece since the change applies to more than just dev environment.